bothan-htx/Cargo.toml

```diff
@@ -7,18 +7,20 @@ license.workspace = true
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+flate2 = "1.0.25"
```

Comment:
please format

Reply:
Please add the dependencies in alphabetical order

Reply:
its in alphabetical order already??

Reply:
or you mean we mix up with the workspace ones? can we just ignore it now then just move some of these that is not in workspace later since it is involved with every sources